### ISSUE TITLE
Syntax errors in redirects

### ISSUE THREAD

**Issue by linawolf** [TYPO3GmbH/site-intercept#106]
@NeoBlack It seems like we have a syntax error in the redirects, see here: https://github.com/TYPO3-Documentation/t3docs-ci-deploy/actions/runs/4055265525/jobs/6978166160

err: nginx: [emerg] unexpected "}" in /etc/nginx/includes/docs_redirects/static_redirects.conf:106
err: nginx: configuration file /etc/nginx/nginx.conf test failed
out: Error while config check.
out: Something went wrong, I do a rollback

I can however find nothing wrong in the file. Could you have a look please?

**Comment by tomasnorre** [comment]
If I do a `nginx -t` directly one the server with the current files present I get 

```shell
nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
nginx: configuration file /etc/nginx/nginx.conf test is successful
```

I cannot see at the first glance what the problem should be. Will try to look into it tomorrow.

**Comment by tomasnorre** [comment]
Is resolved with #107
